crates/autopilot/src/lib.rs

```diff
@@ -439,7 +439,7 @@ pub async fn main(args: arguments::Arguments) {
             .map(|block| {
                 (
                     block.number.expect("number must exist").as_u64(),
-                    block.hash,
+                    block.hash.expect("hash must exist"),
```

Comment:
ubernit: Instead of panic-ing (since it comes from an external API which may have a logic bug that we can't protect against) can we:
```suggestion
                    block.hash.context("hash must exist for mined block")?,
```

Reply:
I find this highly unlikely to fail (since it comes from the client node), but also, we usually use `expect` in `main.rs` file to indicate that app was not able to start with selected arguments (so this change would be against the consistency in `main.rs`). If we change the approach I would rather do it for all possible failures in a separate PR.

Reply:
🤦 - you are right. I saw the file was `lib.rs` and didn't realize this was the autopilot startup code. I agree we can expect here.